lib/commands/init.js

```diff
@@ -98,9 +100,15 @@ module.exports = Command.extend({
 
     await this.runTask('InstallBlueprint', blueprintOpts);
 
+    if (commandOptions.pnpm && !isExperimentEnabled('PNPM')) {
+      throw new Error(`--pnpm support is not enabled by default. Please prefix your command with EMBER_CLI_PNPM=true`);
+    }
+
     if (!commandOptions.skipNpm) {
       let packageManager =
-        commandOptions.yarn === true
+        commandOptions.pnpm === true
```

Comment:
This is getting pretty hard to read; would you mind refactoring it to have fewer ternaries?

Reply:
rogerroger.gif

Reply:
Done :tada: